app/controllers/admin/users_controller.rb

```diff
@@ -13,13 +14,17 @@ def update; end
 
     def destroy
       if @user.destroy
-        redirect_to admin_users_path, notice: "User has been deleted"
+        flash[:notice] = "User has been deleted"
+        @message = flash[:notice]
```

Comment:
No need for this as the flash will go itself with redirect_to. Remove these lines at both places.